Taskfile.yml

```diff
@@ -30,3 +33,19 @@ tasks:
       - bin/**/*
       - provider/cmd/pulumi-resource-nuage/build/**/*
       - provider/cmd/pulumi-resource-nuage/nuage_provider.egg-info/*
+
+  distribute:
+    desc: Create Distribution
+    deps:
+      - build
+    cmds:
+      - make dist
+
+  install-local:
+    desc: Install Project locally
+    deps:
+      - distribute
+    cmds:
+      - make install
+      - pulumi plugin rm resource nuage -y
+      - pulumi plugin install resource nuage ${versionNumber} --file dist/pulumi-resource-nuage-v${versionNumber}-darwin-amd64.tar.gz
```

Comment:
Hard coded Darwin platform... can you fix it somehow ?

Maybe we could agree on using a devcontainer or something ?

Reply:
I think your sh script works but it kinda complex 

We have an example of a working if/else fork in a Dockerfile on another project 

Either way : I approved the PR you can merge whenever you're satisfied with it